clients/mtaclient_v2/mta_v2_client_operations.go

```diff
@@ -0,0 +1,17 @@
+package mtaclient_v2
+
+import (
+	models "github.com/cloudfoundry-incubator/multiapps-cli-plugin/clients/models"
+	strfmt "github.com/go-openapi/strfmt"
+)
+
+// MtaClientOperations drun drun drun x2
+type MtaV2ClientOperations interface {
+	GetMtas(name, namespace *string, spaceGuid string) ([]*models.Mta, error)
+	GetMtasForThisSpace(name, namespace *string) ([]*models.Mta, error)
```

Comment:
Would it be better to be like this:
`GetMtasForNamespace(name, namespace *string) ([]*models.Mta, error)`
Something further, you could try to do some refactoring here which would be to introduce a Builder for the request parameters. Meaning that the code will become:
`GetMtas(MtasRequestParameters parameters) ([]*models.Mta, error)`